pkg/osquery/table/launcher_info.go

```diff
@@ -18,14 +20,22 @@ func LauncherInfoTable() *table.Plugin {
 		table.TextColumn("goos"),
 		table.TextColumn("revision"),
 		table.TextColumn("version"),
+		table.TextColumn("identifier"),
 	}
-	return table.NewPlugin("kolide_launcher_info", columns, generateLauncherInfoTable())
+	return table.NewPlugin("kolide_launcher_info", columns, generateLauncherInfoTable(db))
 }
 
-func generateLauncherInfoTable() table.GenerateFunc {
+func generateLauncherInfoTable(db *bbolt.DB) table.GenerateFunc {
+
 	return func(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+		identifier, err := osquery.IdentifierFromDB(db)
+
```

Comment:
```suggestion
```

Reply:
`kolide_best_practices` doesn't feel like a good choice -- it brings in a lot of weird sub checks, likely filesystem access, and is on the chopping block for future deprecation.

Reply:
@directionless , yep good point ... it failed the CI due to not being able to find a dependent table.

How about kolide_ssh_keys?